testdata/p4_16_samples/strength4-bmv2.p4

```diff
@@ -0,0 +1,74 @@
+#include <core.p4>
```

Comment:
Is this the exact same test as strength4.p4? Can we just rename that file?

Reply:
I remove it.